TechTree/cytoscapeGraph/server.js

```diff
@@ -0,0 +1,27 @@
+//declaration of libraries
```

Comment:
You can eliminate the need for this file.

Instead of doing `node server.js` to serve files in the directory, you can do `python3 -m http.server`

Reply:
`server.js` has been removed.